content/docs/README.md

```diff
@@ -143,6 +143,31 @@ To add a single page <https://example.com/changelog> to the docs sidebar, add th
 
 All available languages for code blocks can be found [here](https://prismjs.com/index.html#supported-languages).
 
+## Images
+
+The images should be sourced in `docs` directory and be used in `.md` with the absolute path
+
+For example:
+
+```md
+├── content
+│ ├── docs
+│ ├── conceptual-guides
+│ ├── images
+│ ├── neon_architecture_2.png // put images in the same directory as your .md file
+│ ├── architecture-overview.md
+```
```

Comment:
```suggestion
```md
├── content
│ ├── docs
│ ├── conceptual-guides
│ ├── images
│     ├── neon_architecture_2.png // put images in the same directory as your .md file
│ ├── architecture-overview.md
```
```